connector.go

```diff
@@ -192,35 +182,43 @@ func (wrb *WapiRequestBuilder) BuildUrl(t RequestType, objType string, ref strin
 	return u.String()
 }
 
-func (wrb *WapiRequestBuilder) BuildBody(obj IBObject) []byte {
-	var jsonStr []byte
+func (wrb *WapiRequestBuilder) BuildBody(t RequestType, obj IBObject) []byte {
+	var objJson []byte
 	var err error
 
-	jsonStr, err = json.Marshal(obj)
+	objJson, err = json.Marshal(obj)
 	if err != nil {
-		log.Printf("Cannot marshal payload: '%s'", obj)
+		log.Printf("Cannot marshal object '%s': %s", obj, err)
 		return nil
 	}
 
-	return jsonStr
+	eaSearch := obj.EaSearch()
+	if t == GET && len(eaSearch) > 0 {
+		eaSearchJson, err := json.Marshal(eaSearch)
+		if err != nil {
+			log.Printf("Cannot marshal EA Search attributes. '%s'\n", err)
+			return nil
+		}
+		objJson = append(append(objJson[:len(objJson)-1], []byte(",")...), eaSearchJson[1:]...)
```

Comment:
nit... and I confess that's how the example I provided was written... but:
`append(objJson[:len(objJson)-1], []byte(",")...)`
can just be:...
`append(objJson[:len(objJson)-1], byte(','))`